internal/pkg/mapper/mapper.go

```diff
@@ -38,6 +39,7 @@ func New(logger *zap.SugaredLogger, fs filesystem.Fs, naming *model.Naming, stat
 	// Mappers
 	m.mappers = append(
 		m.mappers,
+		relations.NewMapper(m.context),
```

Comment:
Pridany novy `mapper`, ktory riesi `relations`.

Doteraz je vyriesene ukladanie/nacitavanie `relations` do `manifest.json`:
https://github.com/keboola/keboola-as-code/blob/4a090b0fbb2af1c6b69f9ec8165ce03626891f65/internal/pkg/model/manifest.go#L63-L68

Je potrebne to teraz prepojit s dalsami castami apky.